config.json

```diff
@@ -1,7 +1,7 @@
 {
   "language": "Dart",
   "active": true,
-  "blurb": "",
+  "blurb": "Dart is an application programming langauge developed by Google which can be used to create web applications and cross-platform mobile applications.",
```

Comment:
I really dislike describing Dart for use only as it relates to web applications and mobile apps.

I would prefer something along the lines of:

> Dart is a general purpose programming language which has seen growth in usage for web applications and cross-platform mobile applications.

Reply:
That's fair. I was trying to come up with a description that demonstrated popular use-cases for Dart. I like your description better. I'll update it when I get a chance.

Reply:
It doesn't have to be exactly like mine, free feel to make it better :)